charts/rasa-x/templates/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.ingress.enabled -}}
+{{- if and .Values.ingress.enabled .Values.nginx.enabled -}}
```

Comment:
Cool, only one small thing. Can you add a comment in the value file that `ingress.enabled` requires `nginx.enabled=true` in order to work?

https://github.com/RasaHQ/rasa-x-helm/blob/967d470db7b530151e29dda71dbbe0c0d092f15e/charts/rasa-x/values.yaml#L401

Reply:
Thanks. I added a small comment :)